pkg/cloud/powervs.go

```diff
@@ -185,7 +181,7 @@ func (p *powerVSCloud) GetPVMInstanceByName(name string) (*PVMInstance, error) {
 		return nil, err
 	}
 	for _, pvmInstance := range in.PvmInstances {
-		if name == *pvmInstance.ServerName {
+		if name == strings.ToLower(*pvmInstance.ServerName) {
```

Comment:
is there a specific reason why we are converting this lower?

Reply:
will ibm cloud respect case in that case? what happens when user creates VM with `sample-vm` and `Sample-vm`

Reply:
probably we can create a request with powervs team to have these constraints integrated as well.